app/src/main/assets/native/ExoPlayerPlugin.js

```diff
@@ -1,73 +1,55 @@
 export class ExoPlayerPlugin {
     constructor({ appSettings, events, playbackManager, loading }) {
-        window['ExoPlayer'] = this;
+        window["ExoPlayer"] = this;
```

Comment:
Why did you change the quotes? It's pretty much a standard to use single quotes in JavaScript

Reply:
That was my VSCode :grimacing: